store/subscription/subscription.go

```diff
@@ -111,3 +114,27 @@ func (t *store) Reset(from, to int64) error {
 	// Delete total balances
 	return t.db.Delete(model.TotalBalance{}, "block_number >= ? AND block_number <= ?", from, to).Error
 }
+
+func (t *store) FindByGroup(groupID int64, query *model.QueryParameters) ([]*model.Subscription, uint64, error) {
+	filter := &model.Subscription{
+		Group: groupID,
+	}
+	db := t.db.Model(&model.Subscription{}).Where(filter)
+	var total uint64
+	err := db.Count(&total).Error
+	if err != nil {
+		return nil, 0, err
+	}
+	start := (query.Page - 1) * query.Limit
+	var result []*model.Subscription
+	err = db.Offset(start).Limit(query.Limit).Order(query.OrderString()).Find(&result).Error
+	if err != nil {
+		return nil, 0, err
+	}
+	return result, total, nil
+}
+
+func (t *store) BatchDelete(groupID int64, addrs [][]byte) (err error) {
+	db := t.db.Where("`group` = ?", groupID).Where("address in (?)", addrs)
```

Comment:
How about use AND? 
```
t.db.Where("`group` = ? AND address in (?)", groupID, addrs)
```

Reply:
`Where ` will combined as `AND`.
So
```
 t.db.Where("`group` = ?", groupID).Where("address in (?)", addrs)
```
is equal to 
```
t.db.Where("`group` = ? AND address in (?)", groupID, addrs)
```